### PR TITLE
🌐 Add Portuguese translation for `docs/tutorial/body.md`

### PR DESCRIPTION
**Pull request by leandrodesouzadev**
This PR translates the tutorial/body.md and add this page to the mkdocs.yml file.

### CODE REVIEW

File: docs/pt/docs/tutorial/body.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+# Corpo da Requisição
+
+Quando você precisa enviar dados de um cliente (Como de um navegador web) para sua API, você envia como um **corpo da requisição**.
```
Comment (Kludex) [inline docs/pt/docs/tutorial/body.md:3]:
```suggestion
Quando você precisa enviar dados de um cliente (como de um navegador web) para sua API, você o envia como um **corpo da requisição**.
```

**Review by cassiobotaro — APPROVED** [review]
LGTM

**Review by diogoduartec — CHANGES_REQUESTED** [review]

File: docs/pt/docs/tutorial/body.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+{!../../../docs_src/body/tutorial004.py!}
+```
+
+Os parâmetros da função serão reconhecidos conforme abaixo:
+
+* Se o parâmetro também é declarado na **rota**, será utilizado como um parâmetro de rota.
+* Se o parâmetro é de um **tipo único** (como `int`, `float`, `str`, `bool`, etc) será interpretado como um parâmetro de **consulta**.
+* Se o parâmetro é declarado como um **modelo Pydantic**, será interpretado como o **corpo** da requisição.
+
+!!! Observação
```
Comment (lsglucas) [inline docs/pt/docs/tutorial/body.md:158]:
```suggestion
!!! note "Observação"
```
Just a small fix

**Review by stlucasgarcia — CHANGES_REQUESTED** [review]

**Review by stlucasgarcia — APPROVED** [review]

**Review by peidrao — APPROVED** [review]

**Review by Riverfount — APPROVED** [review]
Great Job!

**Pull request merged**